spec/workers/search/classified_listing_es_index_worker_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe Search::ClassifiedListingEsIndexWorker, type: :worker, elasticsearch: true do
+  let(:worker) { subject }
+
+  include_examples "#enqueues_on_correct_queue", "high_priority", [1]
+
+  it "raises an error if record is not found" do
+    expect { worker.perform(1234) }.to raise_error(ActiveRecord::RecordNotFound)
+  end
+
+  it "indexes classified_listing" do
+    classified_listing = FactoryBot.create(:classified_listing)
+    expect { classified_listing.elasticsearch_doc }.to raise_error(Elasticsearch::Transport::Transport::Errors::NotFound)
+    worker.perform(classified_listing.id)
+
+    elasticsearch_doc = classified_listing.elasticsearch_doc.dig("_source")
+    cl_serialized_search_hash = classified_listing.serialized_search_hash.deep_stringify_keys
+
+    # To avoid a mess of parsing timestamps in different timezones, just check the key
```

Comment:
I realize this probably isn't great, I'm hoping someone has a better idea. I was trying to avoid manually updating the `bumped_at` timestamp fields on both Hashes. They never match because of differences in timezone. Since we don't _really_ need to compare the values here, I went about this way.

I looked into using some things like `hash_including`, but it got a little tricky because we are comparing nested hashes. Which also makes comparing just the keys more difficult too. From what I've read online, it seems most people implemented their own RSPEC helpers to compare nested hashes (seemed overkill for this). Furthermore, we probably could get away with explicitly checking a subset of key/values (like `id`) and be fine.

It basically came down to update the values to be identical and compare the hashes or compare the keys in the hashes. And I landed here. I'm definitely open to a cleaner way!

Reply:
Would something like `Timecop.freeze("2020-02-18T12:00:00Z"` followed by `Timecop.return` at the end of the spec help? We have several examples in our test suite in case you want to take a look.

Reply:
(partially summarizing a Slack convo with @citizen428)

That's a good call. Funny thing is I actually looked at Timecop docs and didn't realize we already had it in the app (it's Monday 🤦‍♂️).

I looked at this more...we still get slapped with the fact that we have to convert the timestamps and are off by milliseconds. So to go down this road we'd have to round off milliseconds on both the Elasticsearch result and our mappings hash.

I decided to just scrap that approach altogether and explicitly test specific key/value pairs - it feels much better and seems to be more consistent with how we do similar tests in other areas.